test-forms/lakeshorelearning_checkout.html

```diff
@@ -169,8 +169,7 @@
                             <input class="form-control fs-mask" type="text" maxlength="4"
                                    automation-id="co_bill_cvv" aria-required="true" data-bind="value: payments.createPayment().cardVerificationNumber, validateCVV, validateInput:'Num',
 																						valueUpdate: 'keyup', enterkey: function() {$component.enterSaveBillingAddress();}"
-                                   data-manual-scoring="cardSecurityCode"
-                                   style="background-color: rgb(248, 244, 152) !important; color: rgb(51, 51, 51) !important;">
+                                   data-manual-scoring="cardSecurityCode">
```

Comment:
I've just removed the background color. I noticed it while testing. Nothing to see 🚶‍♂️.